src/client/components/Login.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useState } from "react";
+import ReactDOM from "react-dom/client";
+import "../styles/Login.css";
+
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
+import Owner from "./Owner";
+import Createacc from "./Createacc";
+
+interface ErrorMessage {
+  name: string;
+  message: string;
+}
+
+interface FormValue {
+  name: string;
+  pass: string;
+}
+
+function Login() {
+  //seed data
+  const database = [
+    {
+      username: "user1",
+      password: "pass1",
+    },
+    {
+      username: "user2",
+      password: "pass2",
+    },
+  ];
+
+  // React States
+  const [errorMessages, setErrorMessages] = useState<ErrorMessage>({
+    name: "",
+    message: "",
+  });
+  const [loginSuccess, setLoginSuccess] = useState<boolean>(false);
+  const [formValue, setFormValue] = useState<FormValue>({
+    name: "",
+    pass: "",
+  });
+
+  // User Login info
+  const loginErrors = {
+    uname: "invalid username",
+    pass: "invalid password",
+  };
```

Comment:
I think you don't have to define this constant variable. just use the string in the jsx.